app/controllers/manuals_controller.rb

```diff
@@ -125,6 +125,26 @@ def publish
     )
   end
 
+  def discard_draft
+    service = Manual::DiscardDraftService.new(
+      user: current_user,
+      manual_id: manual_id
+    )
+    result = service.call
+
+    if result.successful?
+      redirect_to(
+        manuals_path,
+        flash: { notice: "Discarded draft of #{result.manual_title}" }
+      )
+    else
+      redirect_to(
+        manual_path(manaul_id),
```

Comment:
manual_id*

Reply:
Can we get a test that checks for the unhappy path? We already have two scenarios:
a. a draft manual that has never been published is successfully discarded
b. a draft manual that has been published fails to be discarded

Perhaps a third scenario is necessary: 
c. a draft manual that has never been published that fails to be discarded

As an extra precaution that the redirect works, perhaps you could also check for the the flash message ("Unable to discard draft...") if the action has been unsuccessful? 

I'm pretty sure that's why the "manual_id" typo sneaked through.

Reply:
I don't have any controller specs, because there aren't many in this project, so yeah, there's no sad path test on this.  I'll see if it makes sense to add sad path feature spec or a controller spec.

Reply:
Added a controller spec in a rebase.